src/main/java/edu/cornell/kfs/sys/CUKFSConstants.java

```diff
@@ -151,5 +151,8 @@
     public static final String RIGHT_PARENTHESIS = ")";
     public static final String COLON = ":";
     public static final String SLASH = "/";
+    public static final String BACKSLASH = "\\";
+
+    public static final String CSV_FILE_EXTENSION = ".csv";
```

Comment:
since this is only used by the unit tests should it be in the CUPdpTestConstants?

Reply:
Good point, I will move it to CUPdpTestConstants for now. If we end up needing it for PROD code, then we can add it back in as part of whatever PR requires it.